README.md

```diff
@@ -1220,6 +1225,8 @@ Examples can be found in the [examples](https://github.com/sensu/sensu-puppet/tr
 
 ## Limitations
 
+Changing `sensu::etc_dir` is only supported on systems using systemd.
```

Comment:
This specifically excludes X, Y and Z

Reply:
Do we really want to define which OSes do not support Systemd? I feel like an administrator should know that and puts us in the business of telling admins how their systems work.  I also do not know which OSes we support work with systemd so that will take some time to dig up if we really want that information to be documented.

Reply:
Looks like Ubuntu you can opt-in to systemd on systems that don't use systemd by default, so this gets into some territory where we can't say which OS has systemd because it's really a configuration choice. The only one I can say for sure is EL6 so seems like just saying "systems using systemd" is about as good a description as we can give without potentially falsely labeling some OSes as not supported when they could be supported if configured differently.

Reply:
OK, makes sense.